perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -2668,6 +2917,189 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Checks if these facility, resource, user and member attributes are valid.
```

Comment:
Please change it to "has valid semantics" (also for all other occurrences in this Javadoc).